convert2rhel.ini.example

```diff
@@ -0,0 +1,10 @@
+# -*- coding: utf-8 -*-
```

Comment:
Note: We should have a comment that this file should be installed with mode 0600 since it contains secrets and be sure that we do that in the rpm spec file.

Reply:
That's a good catch! I think that's probably would be a good idea to implement a check in the code as well, so we can be sure that it's correct even after the installation (or manual creation by the user)